client/jetpack-connect/authorize.js

```diff
@@ -594,9 +582,11 @@ export class JetpackAuthorize extends Component {
 
 		recordTracksEvent( 'calypso_jpc_approve_click' );
 
-		if ( 'woocommerce-onboarding' === from ) {
-			recordTracksEvent( 'wcadmin_storeprofiler_connect_store', { use_account: true } );
-		}
+		// Should we keep this around, since we otherwise removed other instances
+		// where we fired this tracks event?
+		// if ( 'woocommerce-onboarding' === from ) {
+		// 	recordTracksEvent( 'wcadmin_storeprofiler_connect_store', { use_account: true } );
+		// }
```

Comment:
@chihsuan just wanted to double-check if we're also ok deleting the firing of this tracks event, and more in general, of the other tracks event across this PR:

- `wcadmin_storeprofiler_create_jetpack_account`
- `wcadmin_storeprofiler_login_jetpack_account`
- `wcadmin_storeprofiler_connect_store`

Reply:
It's ok to delete them. To make sure we don't break any analytics, I searched for them in nosara and didn't find any usage. I also checked the Funnel, and the last usage of the Funnel is 3 years ago before the core profiler was introduced. 

For core profiler, we're using the `calypso_jpc_wc_coreprofiler_*` events for tracking so I don't see the need to keep these.